packages/core/src/Stores/chart-barrier-store.js

```diff
@@ -43,16 +48,18 @@ export class ChartBarrierStore {
         });
 
         this.color = color;
+        this.hideBarrierLine = hideBarrierLine;
+        this.shade = shade;
         this.lineStyle = line_style || BARRIER_LINE_STYLES.SOLID;
         this.onChange = this.onBarrierChange;
 
         // trade_store's action to process new barriers on dragged
         this.onChartBarrierChange =
             typeof onChartBarrierChange === 'function' ? onChartBarrierChange.bind(this) : () => {};
 
-        this.high = +high_barrier || 0; // 0 to follow the price
```

Comment:
@matin-deriv we removed it because barriers will be passed as strings to smartcharts from now on. please refer to https://github.com/maryia-deriv/deriv-app/commit/b315e2d29730f379e0fa0de3b851951ed4f23baa